DuckDuckGo/Preferences/Model/VPNPreferencesModel.swift

```diff
@@ -74,11 +74,15 @@ final class VPNPreferencesModel: ObservableObject {
         }
     }
 
+    var appExclusionsFeatureEnabled: Bool {
+        featureFlagger.isFeatureOn(.networkProtectionAppExclusions)
+    }
+
     /// Whether the excluded sites section in preferences is shown.
     ///
     /// Only necessary because this is feature flagged to internal users.
     ///
-    var showExcludedSites: Bool {
+    var exclusionsAvailable: Bool {
```

Comment:
Whether exclusions are available.  We used this to disable exclusions in App Store for now.  This is currently true only for Sparkle builds.